package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "homepage": "https://nelsinofrancisco.github.io/math_magicians/",
+  "name": "math-magicians",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.14.1",
+    "@testing-library/react": "^11.2.7",
+    "@testing-library/user-event": "^12.8.3",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-scripts": "4.0.3",
+    "web-vitals": "^1.1.2"
```

Comment:
- [ ] Please fix the dependencies  I cant see the landing page of the project 

![klkl](https://user-images.githubusercontent.com/61562626/133141948-a9299f11-b163-4884-a45a-e1deee82a2ea.jpeg)

Reply:
This issue is resolved. Well done!! :clap: :clap: :clap: